build.gradle

```diff
@@ -47,6 +47,14 @@ subprojects {
 
 apply from: 'cppSettings.gradle'
 
+def formatArgs = project.hasProperty('formatArgs') ? project.formatArgs : ''
+
+task format(type: Exec) {
+    workingDir "${rootDir}"
+    commandLine 'python'
+    args 'styleguide/format.py', formatArgs
```

Comment:
Fixed, and this should work on linux now too. @calcmogul could you try linux again?